libs/blocks/editorial-card/editorial-card.css

```
@@ -158,7 +160,8 @@
   display: flex;
   align-items: center;
   justify-content: right;
-  gap: var(--spacing-xxs);
+  column-gap: var(--spacing-s);
+  row-gap: var(--spacing-xs);
```

We could still keep the shorthand
```suggestion
  gap: var(--spacing-xs) var(--spacing-s);
```